.github/workflows/ci.yml

```diff
@@ -36,12 +36,13 @@ jobs:
   build:
     name: ${{ matrix.friendlyName }} ${{ matrix.arch }}
     runs-on: ${{ matrix.os }}
+    container: ${{ matrix.image }}
     permissions:
       contents: write
     strategy:
       fail-fast: false
       matrix:
-        node: [18.14.0]
+        node: [18.16.0]
```

Comment:
I'm curious about this version bump here - do you remember what this helped us with?

Reply:
I just version bumped so that windows and linux versions were close together. they are still part of the stable (ie: no api breakage) node series

Reply:
seems to be a part of the electron bump https://github.com/desktop/desktop/pull/16624

Reply:
The bit that I'm curious about here was the minor version bump in this PR (from `18.14` to `18.16`) - how does this help us out?

Reply:
This appears to be the only version with the `linux-x64-glibc-217` version of Node. Not sure how sustainable this will be in the long-term, mostly wanted to understand the version shift here.

Reply:
see https://github.com/shiftkey/desktop/pull/897#issuecomment-1627812785